quix-frontend/client/src/lib/runner/services/autocomplete/autocomplete-utils.ts

```diff
@@ -44,6 +44,18 @@ export const createMatchMask = (
   return res;
 };
 
+
+export function createCaption(str : string) {
+  const maxCaptionLength = 60;
+  const maxSubCaptionLength = 57;
+
+  if (str.length > maxCaptionLength) {
+    return str.substring(0, maxSubCaptionLength) + "...";
```

Comment:
maybe this can be solved with styles somehow? without changing actual data we gonna display